src/index.html

```diff
@@ -5,8 +5,250 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <header class="header">
+    <div class="header__logo">
+      <a href="#">
+        <img class="header__logo" src="./images/logo.png"
+         alt="moyo logo">
+      </a>
+    </div>
+    <nav class="nav">
+      <ul class="nav__list">
+        <li class="nav__item">
+          <a class="nav__link is-active" href="#apple">Apple</a>
+        </li>
+
+        <li class="nav__item">
+          <a class="nav__link" href="#samsung">Samsung</a>
+        </li>
+
+        <li class="nav__item">
+          <a class="nav__link" href="#smarthphones">Smartphones</a>
+        </li>
+
+        <li class="nav__item">
+          <a class="nav__link" href="#laptops&computers"
+          data-qa="nav-hover">Laptops & Computers</a>
+        </li>
+
+        <li class="nav__item">
+          <a class="nav__link" href="#agadgets">Gadgets</a>
+        </li>
+
+        <li class="nav__item">
+          <a class="nav__link" href="#atablets">Tablets</a>
+        </li>
+
+        <li class="nav__item">
+          <a class="nav__link" href="#photo">Photo</a>
+        </li>
+
+        <li class="nav__item">
+          <a class="nav__link" href="#video">Video</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <body class="body">
```

Comment:
1) Body must be immediately after `</head>` closing tag
2) We don't set class to `<body>` because you can only have one `<body>` on a given page.